curator-operator/base/deploymentconfigs/postgres/deploymentconfig.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
Just noting we have the crunchydata postgres operator on both test and prod clusters now so this, and related manifests, could eventually be replaced using a single PostgresCluster CRD in theory.